packages/nimble/R/genCpp_generateCpp.R

```diff
@@ -187,7 +202,7 @@ cppOutputFor <- function(code, symTab) {
     begin <- nimGenerateCpp(code$args[[2]]$args[[1]], symTab)
     end <- nimGenerateCpp(code$args[[2]]$args[[2]], symTab)
     iterVar <- nimGenerateCpp(code$args[[1]], symTab)
-    part1 <- paste0('for(', iterVar ,'=', begin,'; ', iterVar, '<= static_cast<int>(', end, '); ++', iterVar,')')
+    part1 <- paste0('for(', iterVar ,'=', begin,'; ', iterVar, '<= static_cast<int>(', end, '); ', iterVar,'++ )')
```

Comment:
In the early 2000's pre-increment was cheaper than post-increment, though I'm not sure that's still the case. Is there a good reason to switch from pre- to post- here?